src/compiled/SurveyRunItemView.js

```diff
@@ -404,17 +405,18 @@ SurveyRunItemView = (function(superClass) {
   };
 
   SurveyRunItemView.prototype.onShow = function() {
-    var displayCode, error, error1, message, name, ref;
+    var displayCode, displaycodeFixed, error, error1, message, name, ref;
     displayCode = this.model.getString("displayCode");
     if (!_.isEmptyString(displayCode)) {
+      displaycodeFixed = displayCode.replace("vm.currentView.subtestViews[vm.currentView.index].prototypeView", "Tangerine.progress.currentSubview");
```

Comment:
This is an interesting idea for maintaining backwards compatibility. It does feel kind of hacky but the terms are so unique, collision seems unlikely. I wonder if there is a more transparent place to store this backward compatibility translation.